webpack.config.js

```diff
@@ -17,17 +17,14 @@ module.exports =  {
         publicPath: 'dist/'
     },
     resolve: {
-        extensions: ['.js', '.ts', '.vue'],
-        alias: { //this is needed to get template-compiler?
-            'vue': 'vue/dist/vue.common.js',
-        }
+        extensions: ['.vue', '.ts', '.js']
     },
     module: {
         rules: [
             {
                 test: /\.ts$/,
-                exclude: /node_modules|vue\/src/,

```

Comment:
just exclude `node_modules`, anything else won't be imported as a dependency anyway.